.github/workflows/ci-test.yml

```diff
@@ -0,0 +1,43 @@
+# Github action definitions for unit-tests with PRs.
+
+name: tfma-unit-tests
+on:
+  push:
+  pull_request:
+    branches: [ master ]
+    paths-ignore:
+      - '**.md'
+      - 'docs/**'
+  workflow_dispatch:
+
+jobs:
+  unit-tests:
+    if: github.actor != 'copybara-service[bot]'
```

Comment:
Curious why we skip this on this bot?

Reply:
The testing workflow was configured this way for TFX, and I copied it here.

If we can justify removing that condition, I'm happy to do so.

@embr do you have an opinion here?